### DISCUSSION TITLE
怎么做到输入即翻译，不需要按enter才会翻译？

### DISCUSSION THREAD

**Discussion by OrangeChu** [STranslate/STranslate#179] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/5ef275da-d434-4377-a405-182e90dac217)
想在界面上输入内容后直接翻译，不需要按enter才会翻译。我在设置里有看到类似的选项，但是不是我要的效果，是否我有遗漏什么设置来达到这个效果？谢谢！

**Comment by ZGGSONG** [comment]
没有，你把鼠标放到功能描述旁边的i图标上有更详细的说明

**Comment by ZGGSONG** [comment, marked as answer]
这个功能我回头加一个开关吧

**Reply by OrangeChu** [reply]
OK，十分感谢！

**Comment by OrangeChu** [comment]
@ZGGSONG 你好，我看到新版本有这个功能了。但我打开选项，发现没有效果，我是否遗漏了什么？
![image](https://github.com/user-attachments/assets/065355af-f34e-4bbc-8135-6f8f87249d3c)

**Reply by ZGGSONG** [reply]
随便输入点内容，如果只输入了空格会被过滤

**Reply by OrangeChu** [reply]
https://github.com/user-attachments/assets/7a77521d-be06-4e07-8b2d-802987deef69

我试了也还是不行..我录制了个视频，帮我看一下，谢谢！

**Reply by ZGGSONG** [reply]
> @ZGGSONG 你好，我看到新版本有这个功能了。但我打开选项，发现没有效果，我是否遗漏了什么？
你打开以后有没有保存，需要保存才能生效

**Reply by OrangeChu** [reply]
哦哦！我忘记了，感谢🙏，功能没问题，非常好用！
